metagenomics.py

```diff
@@ -602,11 +616,11 @@ def rank_code(rank):
         return "-"
 
 
-def taxa_hits_from_tsv(f, tax_id_column=3):
+def taxa_hits_from_tsv(f, taxid_column=2):
```

Comment:
The `hits` column in the *.tsv file has changed? Will that cause any issues elsewhere?

Reply:
Nope, it was only used internally anyways